rewriter/tests/simple1/main.c

```diff
@@ -2,6 +2,9 @@
 RUN: cat main.c | FileCheck --match-full-lines --check-prefix=REWRITER %s
 RUN: cat simple1_call_gates_0.ld | FileCheck --check-prefix=LINKARGS %s
 */
+#include <criterion/criterion.h>
+#include <criterion/logging.h>
+#include <criterion/new/assert.h>
```

Comment:
Do we use the new asserts here?

Reply:
Yes, `cr_fatal` is declared in this header.